nengo/connection.py

```diff
@@ -319,9 +317,10 @@ def __init__(self, pre, post, synapse=Default, transform=Default,
         self.function_info = function  # Must be set after transform
         self.solver = solver  # Must be set before learning rule
         self.learning_rule_type = learning_rule_type  # set after transform
-        self.seed = seed
         self.modulatory = modulatory
 
+        super(Connection, self).__init__(label=label, seed=seed)
```

Comment:
No particular reason. The start probably makes more sense in case something goes wrong (as then the label will be set for easier debugging). I'll change!